src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form form--big"
+      action="search"
+      data-qa="big"
+    >
```

Comment:
The class name 'form--big' suggests a modifier in BEM methodology, but there is no corresponding base block. Ensure that class names represent the meaning of the content and not the styles or tag names.